sdk/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltinFun.scala

```diff
@@ -697,6 +698,42 @@ private[lf] object SBuiltinFun {
     }
   }
 
+  final case object SBHexToText extends SBuiltinFun(1) {
+    override private[speedy] def execute[Q](
+        args: util.ArrayList[SValue],
+        machine: Machine[Q],
+    ): Control[Q] = {
+      try {
+        val hexArg = Ref.HexString.assertFromString(getSText(args, 0))
+        val arg = new String(Ref.HexString.decode(hexArg).toByteArray, StandardCharsets.UTF_8)
+
+        Control.Value(SText(arg))
+      } catch {
+        case _: IllegalArgumentException =>
+          Control.Error(
+            IE.Dev(
```

Comment:
IE.Dev sounds like LF-Dev, correct?

Shouldn't these functions go into LF 2.1 ?

Reply:
OK. Sounds fine to me.